image_tools_py/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>image_tools_py</name>
+  <version>0.0.0</version>
+  <description>Python tools to capture/play back images to/from DDS subscriptions/publications.</description>
+  <maintainer email="clalancette@osrfoundation.org">Chris Lalancette</maintainer>
+  <license>Apache License 2.0</license>
+
+  <build_depend>rclpy</build_depend>
+
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
+  <!-- Note that this is only a symbolic dependency on python3-opencv3:
+       it won't actually be resolved by rosdep as the rosdep key does not currently exist -->
```

Comment:
here I would mention that not only the rosdep key but the package itself doesn't exist on any of our supported platforms